test/api/python/integration/test_api_inner.py

```diff
@@ -635,7 +635,7 @@ def test_cortx_update():
     api_call(
         'set_swupdate_repo',
         '1.2.3',
-        source='http://cortx-storage.colo.seagate.com/releases/cortx/integration/centos-7.7.1908/last_successful',  # noqa: E501
+        source='http://<cortx_release_server>/releases/cortx/integration/centos-7.7.1908/last_successful',  # noqa: E501
```

Comment:
@andkononykhin2 , @dmitrydb 
Need your attention here. What could be the best alternative?

Reply:
These tests are outdated and are not run for the moment. So please just add a FIXME. Later we would need an input from @shailesh-vaidya: whether a release repo will be public or not. If not - we will make that thing optional so the test can be run inside Seagate network only.